USAGE.txt

```diff
@@ -1,43 +0,0 @@
-ied is a package manager for Node.
```

Comment:
I've added it in. I'll wait to squash because I don't want to lose context on these diffs.